h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
```

Comment:
One of the things I wanted to avoid is having to fire off 1 SQL query per group, because the `creator` relationship is not loaded. Which plays into the same issue as your comment on `WorldGroup.creator_id`.
I agree that having a `Group.moderators` property is nicer, but is it really worth doing a SQL query for each of the groups on every request to this endpoint?